apps/src/templates/HintDialogContent.jsx

```diff
@@ -0,0 +1,28 @@
+var msg = require('../locale');
+var Lightbulb = require('./Lightbulb.jsx');
+var ReadOnlyBlockSpace = require('./ReadOnlyBlockSpace.jsx');
+
+var HintDialogContent = React.createClass({
+  propTypes: {
+    content: React.PropTypes.string.isRequired,
+    block: React.PropTypes.object,
+  },
+
+  render: function () {
+    var block;
+    if (this.props.block) {
+      block = (<ReadOnlyBlockSpace block={this.props.block} />);
+    }
+
+    return (<div>
```

Comment:
Supernit: My own style preference is to put the opening div on a new line, so that I can easily see how tags match up, ie.:
```
return (
  <div>
    <h3>
    ...
    </h3>
  </div>
);
```
Feel free to ignore or change as you see fit.  If you do decide to change, there are a bunch of other places in this PR you could also make the change.